src/components/ContractDeploy/ContractDeploy.tsx

```diff
@@ -7,95 +7,63 @@ import { setCallResult } from "../../actions";
 import { Button } from "../common/ui";
 
 interface IProps {
-  contractName: string;
   bytecode: any;
   abi: any;
```

Comment:
We should create ABI type from the following spec.

* https://solidity.readthedocs.io/en/v0.7.2/using-the-compiler.html#output-description
* https://solidity.readthedocs.io/en/develop/abi-spec.html#json

Reply:
Import types from:

https://github.com/ethereum/remix-plugins-directory/blob/master/common-api/compiler/type/output.ts